examples/create_motor_group.py

```diff
@@ -37,10 +39,10 @@
     )
     # Print out all currently set instance attributes
     for attr, val in motor_group.__dict__.items():
```

Comment:
This could be a future action item:
Can we implement some feature in the MotorGroup class such that calling `print(motor_group)` or `pprint(motor_group)` performs the same feature?

Reply:
Feel free to create a issue/feature request for this. We can work on this later.

Reply:
Great idea!

Reply:
Added a TODO (which hopefully turns into an issue after this merge is complete)